//
// Copyright 2018 Asylo authors
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//

syntax = "proto2";

package asylo;

import "asylo/enclave.proto";

extend EnclaveInput {
  // The address of the server to contact.
  optional string server_address = 201042554;

  // The input to the Hello RPC.
  optional string rpc_input = 201042555;
}

extend EnclaveOutput {
  // The response from the gRPC server.
  optional string rpc_result = 200746417;
}
